stable/redis/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: redis
-version: 10.0.1
+version: 10.0.2
```

Comment:
I would bump the minor version, as it is an extra feature

Reply:
Done :D